frontend/src/styles.css

```diff
@@ -18,4 +18,14 @@ body, html {
 
 .card {
     margin-bottom: 10px;
+}
+
+.bootstrap-grid {
+    display: flex;
+    min-height: 100vh;
+    flex-direction: column;
+}
```

Comment:
The main style.css should not have this specific css, this should be done in the places where it is used